Vindicta.Altis/Templates/initFactions.sqf

```diff
@@ -29,4 +31,4 @@
 ["Templates\Factions\RHS_LDF_ranger.sqf", T_FACTION_Police]			call t_fnc_initializeTemplateFromFile;
 ["Templates\Factions\RHS_LDF_NATO_police.sqf", T_FACTION_Police] 	call t_fnc_initializeTemplateFromFile;
 ["Templates\Factions\WW2_Heer_police.sqf", T_FACTION_Police]		call t_fnc_initializeTemplateFromFile;
-["Templates\Factions\WW2_UK_police.sqf", T_FACTION_Police]			call t_fnc_initializeTemplateFromFile;
```

Comment:
Looks like end of file newline was removed here? Perhaps by accident but it is good practice to end file with newline.